spec/controllers/teachers/school_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe Teachers::SchoolsController, type: :controller do
+
+  let(:school)          { create(:school) }
+
+  describe "GET #show" do
+
+    context "school not in cache" do
+        before(:each) do
+          allow(AggregateSchoolService).to receive(:caching_off?).and_return(false, true)
+          allow_any_instance_of(AggregateSchoolService).to receive(:aggregate_school).and_return(school)
+          sign_in_user(:school_admin, school.id)
+        end
+
+        it "shows error page" do
```

Comment:
Does this show an error page, or show the normal page with 0 readings..?

Reply:
This isn't the show page, its a separate template that catches situation when school isn't in cache.